sdk/python/kubeflow/training/api/py_torch_job_client.py

```diff
@@ -15,67 +15,147 @@
-                 %s\n" % e)
-
-        return outputs
+                 %s\n"
+                % e
+            )
+
+        logging.info("PyTorchJob {} has been created".format(pytorchjob.metadata.name))
+
+    def create_pytorchjob_from_func(
+        self,
+        name: str,
+        func: Callable,
+        parameters: Dict[str, Any] = None,
+        base_image: str = constants.PYTORCHJOB_BASE_IMAGE,
+        namespace: str = utils.get_default_target_namespace(),
+        num_worker_replicas: int = None,
+        packages_to_install: List[str] = None,
+        pip_index_url: str = "https://pypi.org/simple",
+    ):
+        """Create PyTorchJob from the function.
+
+        Args:
+            name: Name for the PyTorchJob.
+            func: Function that PyTorchJob uses to train the model. This function
+                must be Callable. Optionally, this function might have one dict
+                argument to define input parameters for the function.
+            parameters: Dict of input parameters that training function might receive.
+            base_image: Image to use when executing the training function.
+            namespace: Namespace for the PyTorchJob.
+            num_worker_replicas: Number of Worker replicas for the PyTorchJob.
+                If number of Worker replicas is 1, PyTorchJob uses only
+                Master replica.
+            packages_to_install: List of Python packages to install before
+                executing the training function.
+            pip_index_url: The PyPI url from which to install Python packages.
+        """
 
-    def get(self, name=None, namespace=None, watch=False,
-            timeout_seconds=600):  # pylint: disable=inconsistent-return-statements
+        # Check if at least one worker replica is set.
+        if num_worker_replicas is None:
+            raise ValueError("At least one Worker replica for PyTorchJob must be set")
+
+        # Check if function is callable.
+        if not callable(func):
+            raise ValueError(
+                f"Training function must be callable, got function type: {type(func)}"
+            )
+
+        # Get PyTorchJob Pod template spec.
+        pod_template_spec = utils.get_pod_template_spec(
+            func=func,
+            parameters=parameters,
+            base_image=base_image,
+            container_name="pytorch",
+            packages_to_install=packages_to_install,
+            pip_index_url=pip_index_url,
+        )
+
+        # Create PyTorchJob template.
+        pytorchjob = models.KubeflowOrgV1PyTorchJob(
+            api_version=f"{constants.PYTORCHJOB_GROUP}/{constants.PYTORCHJOB_VERSION}",
+            kind=constants.PYTORCHJOB_KIND,
+            metadata=client.V1ObjectMeta(name=name, namespace=namespace),
+            spec=models.KubeflowOrgV1PyTorchJobSpec(
+                run_policy=models.V1RunPolicy(clean_pod_policy=None),
+                pytorch_replica_specs={},
+            ),
+        )
```

Comment:
This is just a question, did we not support [ElasticPolicy](https://github.com/kubeflow/training-operator/blob/2728b25254d4840661b8fdb34c295893cc6de7cb/sdk/python/kubeflow/training/models/kubeflow_org_v1_elastic_policy.py) since before in pytorch python client?

If we don't support ElasticPolicy now, should we add the description to README about it?

Reply:
@tenzen-y Yes, I think we don't support it now. We should have discussion what is the best and easy way to introduce ElasticPolicy to PyTorch SDK. @tenzen-y @johnugeorge @anencore94 Should we create an issue to track it ?

Reply:
> Yes, I think we don't support it now.

Thanks for clarifying.

> Should we create an issue to track it ?

I think that we need to create the issue to follow this in another place.